Forensics-OSINT/_posts/2020-07-14-Insanity-Check.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Insanity-Check"
```

Comment:
You can remove the `-` from the title if you want

Reply:
Yup